spec/support/helpers.rb

```diff
@@ -158,6 +162,12 @@ class ::Plain < ActiveRecord::Base
         class ::Event < ActiveRecord::Base
           extend ChronoModel::TimeMachine::TimeQuery
         end
+
+        class ::Animal < ActiveRecord::Base
+          include ChronoModel::TimeMachine
+        end
+
+        class ::Dog < Animal; end
```

Comment:
I was expecting a 🐐 somewhere and instead I got a 🐕

Reply:
![TROLOLOL](https://media0.giphy.com/media/6XjF2YYHFTkWI/giphy.gif)

ref: https://www.youtube.com/watch?v=2Z4m4lnjxkY

Reply:
You have your Goat now: [🐐](https://github.com/ifad/chronomodel/pull/86/files#diff-b1e066867ad264cea50a52d721ed0d49R18)